app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,13 @@
+class CommentsController < ApplicationController
+def index
+end
+def new
+@comment=Comment.new
+end
+def create
+  @comment = Comment.new(params[:comment])
+ 
+  @comment.save
+  redirect_to @comment
```

Comment:
pehele to nomenclature seekh
comment wali html kya hota h?
new comment link kunl jaye bol